lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -170,36 +170,6 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
               <span>Fork</span>
             </button>
           </.menu_item>
-          <span
-            class="tooltip left"
-            data-tooltip={session.file == nil && "Save this notebook before starring it"}
-          >
-            <.menu_item disabled={session.file == nil}>
-              <%= if notebook_starred?(session, @starred_notebooks) do %>
```

Comment:
We can remove `notebook_starred?` and also the event handlers for `"(un)star_notebook"` :)